### DISCUSSION TITLE
Issues for the Calculation of Bayes Factors

### DISCUSSION THREAD

**Discussion by jhmaindonald** [easystats/easystats#225] [General]
What has guided the choice of a normal prior as the default, mostly, in the vignettes? Why prefer it to use of a Cauchy prior? As implemented in the BayesFactor package (Morey and Rouder 2018), this uses a numerical approximation that avoids the convergence issues that arise for Markov Chain Monte Carlo. I cannot see why one would want to use Cauchy as implemented in bayestestR (Makowski, Ben-Shachar, and Lüdecke 2019). I would use bayestestR with a Normal prior only if provided with a good reason to prefer, in a specific analysis, that to Cauchy. What reasons might be adduced? The choice of prior, and issues with convergence, appear to me to trump differences that arise from different approaches to scaling. Accordingly, I regard these as secondary.

The attached pdf pursues these and related issues.
[Issues for the Calculation of Bayes Factors.pdf](https://github.com/easystats/easystats/files/8649755/Issues.for.the.Calculation.of.Bayes.Factors.pdf)
.

**Comment by mattansb** [comment]
Again, this is an issue of scope. `bayestestR` itself does *not* fit the models, set priors or sample from posteriors -- these are up to the user with which ever package they might want, with whatever defaults (or not) they use. Instead, `bayestestR` extracts information from the (supported) fitted models, to facilitate post-estimation analyses - describing posteriors, ROPE, Bayes factors, etc. 

The "reasoning" behind the use of normal priors is... low friction for the reader, as those are not really the focus of the vignettes.

To be clear, the vignettes are not fully fledged guides for model fitting with Stan or BayesFactor, but are guides for the functionality offered by bayestestR.

---

You seem to be under the impression that the difference between our examples and `BayesFactor` are the *shape* and scale of the priors, but these are rather minor differences compared to the issue of **parameterization**. See https://doi.org/10.31234/osf.io/kxhfu and https://doi.org/10.1002/sim.9278 which discuss these points further.

**Reply by jhmaindonald** [reply]
From a quick look, the second article at least appears to have the sort of commentary that I am looking for.   In part, what I am doing is putting out what are perhaps naive questions for those who are firmly homed in the bayesttestR Bayesian world, in the attempt to find guidance for myself and others who find it much less familiar.

**Reply by jhmaindonald** [reply]
You say that the vignettes "are guides for the functionality offered by bayestestR".
Some of the titles appear, certainly on first blush, to offer more than that.  I'd
certainly expect that a Bayesian master would have wrestled with the sorts of
questions that I have asked.
  2. Confirmation of Bayesian skills
  3. Become a Bayesian master
  o Reporting guidelines

Are you aware of a mailing list or suchlike that is directed to the whole range
of Bayesian packages that are available in R, and maybe to discussing what
fits where in a larger universe?

The first of the articles that you reference is not easy to get a handle on.  
The meta-analysis context of the second is one where it is reasonably 
possible to check out and compare priors, in a way that is not possible in 
the analysis of any individual study.  However, the major issue is, surely, 
whether or not to transform proportions before priors enter the picture.
